src/components/Tag.js

```diff
@@ -75,27 +98,23 @@ const Tag = ({ color, disableMenu, slug, text, title, maxValue, ...props }) => {
             '?search[stringTraits][0][name]=' +
             title +
             '&search[stringTraits][0][values][0]=' +
-            text
+            originalValue ?? text
```

Comment:
AFAI understa the logic of `transformUniqueAssetTraitsForPresentation`, there's no case when `originalValue` is not defined UNLESS `value` is not defined. However, if `value` is not defined` then we do filtering  in `UniqueTokenAttributes`
```suggestion
            originalValue
```

Reply:
I have one fundamental question, is Tag.js going to be used anywhere else than in this screen? Given it is in `src/components` and not in `src/components/unique-token` it was likely intended to be general. I didn't rewrite it in TypeScript but I would love `originalValue` to be optional in Tag. If you pass anything there and just want to use this tag anywhere, you may not need to map anything and may not need to use `originalValue` at all. So when `originalValue` us `undefined` we can just use what someone provided as a general `text` prop.
If you think that my thinking here makes no sense, I will gladly move Tag.js to `src/components/unique-token` and make it very specific for unique token view case.
If you see any logic in my thinking I would prefer to still leave it, I can document it with a comment that `originalValue` is optional or just try to rewrite it in TypeScript if that makes sense

Reply:
Our file structure is problematic and we don't really care about this. It's hard to refactor this. If you want to move this, go for it. 
We have no plans to use it somewhere else. Also, I am not a fan of programming for the future.

Reply:
I will make it local then and always use originalValue since it will always be there given the logic of everything else